test/cppsuite/configs/config_poc_test_default.txt

```diff
@@ -15,6 +15,11 @@ runtime_monitor=
     (
         enabled=true,
         limit=100
+    ),
```

Comment:
config_poc_test_stress.txt should be updated too

Reply:
What do you expect the value to be? The only reason I added this was to check whether it works so I just put it high enough that it stopped triggering the check.

Reply:
As discussed, can be removed for now.

Reply:
3d350ae